docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -1,36 +1,40 @@
-# Developing a desktop viewer
+# Developing a iTwin Desktop Viewer
 
 ## Setup
 
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
+- [Install necessary prerequisites]($docs/learning/tutorials/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`
+  - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code
+- Follow these steps to obtain a new OIDC client to use in your viewer application:
+  1. Go to <https://developer.bentley.com>
+  2. Click the **Sign In** button and sign-in using your Bentley account credentials
+      - If you have not already registered, click **Register now** and complete the registration process.
+  3. Click on your user icon and navigate to the **My Apps** page
+  4. Click the **Register New** button
+  5. Give your application a Name
+  6. Select the **Visualization** API
+  7. Select application type **Desktop**
+  8. Enter **Redirect URL** `http://localhost:3000/signin-callback`
+  9. Enter **Post logout Redirect URL**: `http://localhost:3000`.
+  10. Click the **Save** button
 
-## Create application
+## Build
 
-- Navigate to [developer.bentley.com](https://developer.bentley.com/register/)
-- Sign in
-- Configure application
-  - Provide application name
-  - API associations: "Visualization"
-  - Application type: "Desktop/Mobile"
-  - Redirect URI: `http://localhost:3000/signin-callback`
-  - Post logout redirect URI: `http://localhost:3000`
-  - Save
-- Click on the name of the client you just created to see the details
-- Copy Client ID to your clipboard
+- From a terminal at your application's root directory, `npm run build`. This will create a deployment-ready build in the "build" folder within the application's root directory. It is not necessary to build the application during development.

```

Comment:
idk how i feel about this and saying its "deployment ready"
build comand will only build the frontend, still need to build the backend part

Reply:
Does the `npm run build` command of the viewer not build both... I thought it was setup to do `npm run build:backend && npm run build:frontend`? If  not, I'd want to revisit that.

Reply:
right now it just builds frontend, but yea we'll want to update the template, ill make a note to do that
![image](https://user-images.githubusercontent.com/11051042/130271973-4e6c8fc9-284d-45fd-aec1-3e68b92c44ad.png)

Reply:
but still, we'd need to bundle both the frontend build (build output dir) and backend build (lib output dir)

Reply:
Yes, so if you run the both of those commands and then the dev runs `npm run electron` that should start their app in non-dev mode.

We do have the backend-webpack-tools but I don't think we want to promote those too much yet in the Electron case.